extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -163,6 +163,56 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
     });
   });
 
+  describe('amp-live-list integration', () => {
+    it('should detect bindings in initial live-list elements', () => {
+      const liveListItems = fixture.doc.getElementById('liveListItems');
+      expect(liveListItems.children.length).to.equal(1);
+
+      const liveListItem1 = fixture.doc.getElementById('liveListItem1');
+      expect(liveListItem1.firstElementChild.textContent).to.equal('unbound');
+
+      const button = fixture.doc.getElementById('changeLiveListTextButton');
+      button.click();
+      return waitForBindApplication().then(() => {
+        expect(liveListItem1.firstElementChild.textContent).to
+            .equal('hello world');
+      });
+    });
+
+    it('should apply scope to bindings in new list elements', () => {
+      const liveList = fixture.doc.getElementById('liveList');
+      const liveListItems = fixture.doc.getElementById('liveListItems');
+      expect(liveListItems.children.length).to.equal(1);
+
+      const liveListItem1 = fixture.doc.getElementById('liveListItem1');
+      expect(liveListItem1.firstElementChild.textContent).to.equal('unbound');
+
+      const impl = liveList.implementation_;
+      const update = document.createElement('div');
+      update.innerHTML =
+          `<div items>` +
+          ` <div id="liveListItem2" data-sort-time=${Date.now()}>` +
+          `    <p [text]="liveListText">unbound</p>` +
+          ` </div>` +
+          `</div>`;
+      impl.update(update);
+      fixture.doc.getElementById('liveListUpdateButton').click();
+
+      let liveListItem2;
```

Comment:
Sorry I meant "after" the live list update. Maybe you can ID one live list item "pizza" and another "bacon" to go with the theme of the example... up to you though.